notifications_python_client/base.py

```diff
@@ -65,23 +65,29 @@ def request(self, method, url, data=None, params=None):
 
         logger.debug("API request {} {}".format(method, url))
 
-        payload = json.dumps(data)
-
         api_token = create_jwt_token(
             self.api_key,
             self.service_id
         )
 
+        kwargs = {
+            "headers": self.generate_headers(api_token),
+        }
+
+        if data:
```

Comment:
I'd go for `is not None` for these checks. We're unlikely to ever use `0` or `False` as the payload, but you never know.

Reply:
Good point, done and rebased 👍